pulsar/consumer_multitopic.go

```diff
@@ -160,3 +160,14 @@ func (c *multiTopicConsumer) Close() {
 		close(c.closeCh)
 	})
 }
+
+func (c *multiTopicConsumer) Seek(msgID MessageID) error {
```

Comment:
We should return an error here and not allow Seek on multi topic consumer since it does not make sense. Same for the regex consumer.

Reply:
@wangjia007bond  agree with @cckellogg .

Note: this operation can only be done on non-partitioned topics. For these, one can rather perform the seek() on the individual partitions.

Reply:
yes, make sense, this particularly msg id actually can't applied to all the topics for multtopic consumer or regex consumer.

Reply:
Done.